src/Sulu/Bundle/MediaBundle/Admin/MediaAdmin.php

```diff
@@ -176,6 +184,18 @@ public function configureViews(ViewCollection $viewCollection): void
                         ->setParent(static::EDIT_FORM_VIEW)
                 );
             }
+
+            if ($this->referenceViewBuilderFactory->hasReferenceListPermission()) {
```

Comment:
check if we move this into Activity Tab as a subtab

Reply:
Moved into activity tab